mmv1/templates/terraform/custom_expand/sd_full_url.tmpl

```diff
@@ -4,7 +4,7 @@ func expand{{$.GetPrefix}}{{$.TitlelizeProperty}}(v interface{}, d tpgresource.T
   } else if strings.HasPrefix(v.(string), "https://") {
     return v, nil
   }
-  url, err := tpgresource.ReplaceVars(d, config, "{{"{{"}}ServiceDirectoryBasePath{{"}}"}}" + v.(string))
+  url, err := tpgresource.ReplaceVars(d, config, "https://servicedirectory.googleapis.com/v1/" + v.(string))
```

Comment:
Confirmed by service team member @erikmej-code, `https://servicedirectory.googleapis.com/v1/` is always used even if `v1beta` is explicitly sent

Reply:
can we comment that in the code itself?